lib/game/Minesweeper/components/cellwidget.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import '../management/mineboard.dart';
+import '../management/gamelogic.dart';
+import '../theme/colors.dart';
+
+Widget widgetFlag({required bool visible}){
```

Comment:
It should be a StatelessWidget instead of a top-level function, for example:
```dart
class CellFlag extends StatelessWidget {
  // complete this widget.
}
```

Reply:
Yes I do.